src/main/java/io/aiven/connect/jdbc/sink/JdbcDbWriter.java

```diff
@@ -34,11 +35,17 @@
 import org.slf4j.LoggerFactory;
 
 public class JdbcDbWriter {
+
     private static final Logger log = LoggerFactory.getLogger(JdbcDbWriter.class);
 
+    private static final Pattern NORMALIZE_TABLE_NAME_FOR_TOPIC = Pattern.compile("(?<!^)[^a-zA-Z0-9_-]");
```

Comment:
I still have a couple of comments about the regular expression:
1. TBH I still don't get the point of this lookback `(?<!^)`. It means "if the previous character is not the beginning of the string". With it, for example, `...` (a valid Kafka topic name) will be converted into `.__` instead of `___` when `table.name.format=${topic}` (the default). Am I missing something and the lookback is still needed?
2. Speaking of `-`, [this comment](https://github.com/aiven/aiven-kafka-connect-jdbc/pull/37/#discussion_r440181400) is right to note that `-` is not a valid part of an SQL identifier so should be replaced as well.